gsplat/cuda/csrc/bindings.h

```diff
@@ -213,4 +213,11 @@ fully_fused_projection_packed_bwd_tensor(
     const torch::Tensor &v_depths,                      // [nnz]
     const torch::Tensor &v_conics,                      // [nnz, 3]
     const at::optional<torch::Tensor> &v_compensations, // [nnz] optional
-    const bool viewmats_requires_grad, const bool sparse_grad);
\ No newline at end of file
+    const bool viewmats_requires_grad, const bool sparse_grad);
+
+std::tuple<torch::Tensor, torch::Tensor> ComputeRelocationCUDA(
```

Comment:
Could you change the name of this to the same pattern with the existing functions? I know the pattern we have is not a standard pattern but let's just keep them consistent for now.

Reply:
Done.